### PR TITLE
Updated Terminal.tsx and Terminal.scss

### LINKED ISSUE #164: Minor Update: Copy-to-clipboard button for code snippets
Something useful for users could be a copy-to-clipboard button in the top-right corner of each code snippet. This youtube video gives a great guide on how to accomplish this with react-syntax-highlighter -- which is also what we're using for the Terminal component. 

https://www.youtube.com/watch?v=iAAOvv_4ONI&ab_channel=%C3%87elikK%C3%B6seo%C4%9Flu

Reference Image:
image

Current Terminal Component:
image

### PR DESCRIPTION
**Pull request by lawtlee**
Added a copy button for code snippets.

## Summary

Closes #164 

- Added copy button to code snippets
- Added two dependencies:
  - react-copy-to-clipboard
  - react-icons

## Screenshots

https://github.com/uclaacm/parcel-pointers/assets/104863284/4b4e002f-2222-4cbd-b055-b01f55e42141

### CODE REVIEW

File: src/styles/Terminal/Terminal.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+  position: relative;
   width: 80%;
 }
+
+.copybutton {
+  position: absolute;
+  right: 14px;
+  top: 14px;
+
+  &:hover {
```
Comment (Dankco) [inline src/styles/Terminal/Terminal.scss:13]: Can you make the hover state also set the cursor to be a pointer? Otherwise, looks good!

**Review by Dankco — COMMENTED** [review]
I didn't realize that alyssa actually put a guide to this on the ticket- always thought it would require some research, but good job!

File: src/styles/Terminal/Terminal.scss (hunk 1 above)
Comment (lawtlee) [reply src/styles/Terminal/Terminal.scss:13]: https://github.com/uclaacm/parcel-pointers/assets/104863284/ab7d1e01-b9b1-4d58-8e01-5a3e14d085b8

**Review by Dankco — APPROVED** [review]
Looks good!

**Pull request merged**